pkg/beyla/config.go

```diff
@@ -262,7 +262,11 @@ func (c *Config) Validate() error {
 	}
 
 	if !c.Enabled(FeatureNetO11y) && !c.Enabled(FeatureAppO11y) {
-		return ConfigError("missing to enable application discovery or network metrics. Check documentation")
+		return ConfigError("at least one of 'network' or 'application' features must be enabled. " +
+			"Enable Otel export features using the 'OTEL_EBPF_OTEL_METRIC_FEATURES=network,application' environment variable " +
```

Comment:
small nit, we should write `Otel` as `OTel` or use the full word, `OpenTelemetry`.

Reply:
np, fixed in 00b7acd7